web/frontend/src/components/navigation/hooks/use-base-pose.ts

```diff
@@ -0,0 +1,47 @@
+import type { ServiceError } from '@viamrobotics/sdk';
+import type { GeoPose } from '@viamrobotics/prime-blocks';
+import { useNavClient } from './use-nav-client';
+import { writable, get } from 'svelte/store';
+import { setAsyncInterval } from '@/lib/schedule';
+import { useDisconnect } from '@/hooks/robot-client';
+import { useMemo } from '@/lib/use-memo';
+
+export const useBasePose = (name: string) => {
+  return useMemo(() => {
```

Comment:
Unless I'm reading this incorrectly, I don't think this `useMemo` is doing anything. Every time you call `useBasePose`, a new arrow function will be created, so it will look like a new map entry to `useMemo` as written, and nothing will be memoized

Reply:
Gah you're right 🤦 Should've known better than to write this on a Friday afternoon. It's basically a no-op 😂 . Gonna remove this now.